src/index.ts

```diff
@@ -15,12 +15,15 @@ export const angularModule = angular.module(name, [])
           popgun.init();
         },
 
-        init: function($scope, $element): void {
-          $element[0].addEventListener('PopgunContentSetup', function(e) {
-            let pop = popgun.getPopFromGroupId((<Element>e.target).getAttribute('popgun-group'));
-            $compile(pop.popOver.element)($scope);
-            $scope.$apply();
-          }, false);
+        init: function($scope, el): void {
+          if (!el.hasAttribute('popgun-listening')) {
+            el.addEventListener('PopgunContentSetup', function(e) {
```

Comment:
If someone is trying to initialize a popgun that is already initialized, that's an error right? Can we throw an error in the else case? Or some kind of indicator that this is a no-no?